for-posdealers/04-after-sales/troubleshooting-tse-as-a-service.md

```diff
@@ -1,12 +1,30 @@
 # Troubleshooting TSE-as-a-Service
 
+## Hardware TSE-as-a-Service
+
+### Subscription
+
 Bei einer Verlängerung der Subscription, oder falls die zur Verfügung stehenden Signaturen der Hardware-TSE aufgebraucht wurden, erfolgt derzeit keine automatische Lieferung einer neuen Hardware TSE durch fiskaltrust. Der Tausch muss wie unten beschrieben über den fiskaltrust Händlersupport beantragt werden.
 
 Die zur Verfügung gestellte Hardware TSE bleibt im Eigentum der fiskaltrust Germany GmbH und ist bei Tausch bzw. bei Ersatz sowie bei Vertragsende an fiskaltrust zurück zu senden.
 
-#### Antrag auf Tausch einer Hardware TSE
+### Störungsmeldungen
+
+fiskaltrust leitet jene Störungsmeldungen von Kassenbetreiber an den verbunden Kassenhändler weiter, bei welchen der Kassenhändler die Möglichkeit hat bzw. es ihm zumutbar ist diese zu beheben. 
+
+### Hardwareaustausch 
+
+Für den Austausch einer Hardware-TSE im Rahmen von Garantie oder Gewährleistung oder für einen außerordentlichen Austausch im Rahmen der [Fair-Use-Bedingungen](../../for-posoperators/market-de-fair-use-policy.md) muss als erster Schritt eine Ticket-Nummer per E-Mail unter [support@fiskaltrust.de](mailto:support@fiskaltrust.de") angefragt werden. Rücksendungen müssen in den Empfängerdaten diese Ticket-Nummer enthalten. Bei Fällen, in denen die Rücksendung nach hin Sendung einer Vorabaustausch-Hardware-TSE erfolgt, muss die Rücksendung innerhalb von 14 Tagen bei fiskaltrust einlangen, ansonsten wird für das Vorab-Austauchgerät der reguläre Shop-Preis berechnet. 
```

Comment:
```suggestion
Für den Austausch einer Hardware TSE im Rahmen von Garantie oder Gewährleistung oder für einen außerordentlichen Austausch im Rahmen der [Fair-Use-Bedingungen](../../for-posoperators/market-de-fair-use-policy.md) beachten Sie bitte das [Service Level Agreement von fiskaltrust](../../for-posoperators/service-level-agreement.md) 
```
Der Text sollten nicht redundant zur SLA sein.
Die SLA sind derzeit noch nicht hier verfügbar: (../../for-posoperators/service-level-agreement.md) --- Der Link sollte auf den zukünftigen Speicherort verweisen.